components/base/properties.go

```diff
@@ -0,0 +1,36 @@
+// Package base contains an enum representing optional base features
+package base
+
+import pb "go.viam.com/api/component/base/v1"
+
+// Properties is an structure representing optional base features
+// of a base reporting its respective minimum turning radius and
+// width, in units of meters (M).
```

Comment:
I'm open to disagreement on this, but I think this comment should be shortened. Saying it reports its respective minimum turning radius and width in units of meters does not add anything I couldn't easily get from the code itself (there are two variables, `TurningRadiusMeters` and `WidthMeters`, and it's already clear what they represent).